src/plot_splines.jl

```diff
@@ -0,0 +1,110 @@
+using BSplineKit, Unfold
+"""
+    plot_splines(m::UnfoldModel; kwargs...)
+    plot_splines!(f::Union{GridPosition, GridLayout, Figure}, m::UnfoldModel; kwargs...)
+
+Shows two subfigures. First, baseses of splines. Second, density of splines.\\
+Dashed lines shows spline knots.
+
+## Arguments:
+
+- `f::Union{GridPosition, GridLayout, Figure}`
+    `Figure`, `GridLayout`, or `GridPosition` to draw the plot.
+- `m::UnfoldModel`\\
+    UnfoldModel with splines.
+$(_docstring(:splines))
+
+**Return Value:** `Figure` with splines and their density for basis functions.
+"""
+plot_splines(m::UnfoldModel; kwargs...) = plot_splines(Figure(), m; kwargs...)
+
+function plot_splines(f::Union{GridPosition,GridLayout,Figure}, m::UnfoldModel; kwargs...)
+    config = PlotConfig(:splines)
+    config_kwargs!(config; kwargs...)
+    ga = f[1, 1] = GridLayout()
+
+    terms = Unfold.formulas(m)[1].rhs.terms
+    spl_title = join(terms, " + ")
+
+    splFunction = Base.get_extension(Unfold, :UnfoldBSplineKitExt).splFunction
+    spl_ix = findall(isa.(terms, Unfold.AbstractSplineTerm))
+
+    spline_terms = [terms[i] for i in spl_ix]
+    subplot_id = 1
```

Comment:
maybe an assert if no spline_term has been found?

Reply:
assert what?
P.S:. all other comments implemented

Reply:
assert that spl_ix is not empty :)